libs/utils/src/votes.ts

```diff
@@ -84,3 +88,28 @@ export const buildVoteFromCvr = ({
 
   return vote
 }
+
+/**
+ * Gets all the vote options a voter can make for a given yes/no contest.
+ * @returns ContestVoteOption[] ex. ['yes', 'no']
```

Comment:
This JSDoc line doesn't really [add any information](https://google.github.io/styleguide/tsguide.html#redundant-comments) and [shouldn't specify the type](https://google.github.io/styleguide/tsguide.html#omit-comments-that-are-redundant-with-typescript).